### PR TITLE
added authorization service to the chart

### PR DESCRIPTION
**Pull request by thrineshactian**

### CODE REVIEW

**Comment by thrineshactian** [comment]
related PRs : https://github.com/actian-corporation-it/datacloud-gitops-config/pull/151  , https://alm.actian.com/bitbucket/projects/DF/repos/integration-manager-kubernetes/pull-requests/156/overview, https://alm.actian.com/bitbucket/projects/DF/repos/integration-manager-stack/pull-requests/186/overview

File: templates/authorizationService-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        - name: AWS_SECRET_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              key: aws-secret-key
+              {{- if .Values.existingSecret }}
+              name: {{ .Values.existingSecret }}
+              {{- else }}
+              name: {{ template "integration-manager.fullname" . }}
+              {{- end }}
+        {{- end }}
```
Comment (auswells) [inline templates/authorizationService-deployment.yaml]: I don't think you need the AWS env block (lines 58-77)

Comment (thrineshactian) [reply templates/authorizationService-deployment.yaml]: updated

**Comment by thrineshactian** [comment]
2022-08-25 10:37:29.023 DEBUG 1 --- [tp1944815218-19] c.a.d.o.s.controller.LoginController     : authMethod headers ==>[Lorg.apache.http.Header;@44348f38
2022-08-25 10:37:29.028 DEBUG 1 --- [tp1944815218-18] c.a.d.o.server.lockout.LockoutService    : Successful Login Event for [org.springframework.security.core.userdetails.User [Username=integration-manager, Password=[PROTECTED], Enabled=true, AccountNonExpired=true, credentialsNonExpired=true, AccountNonLocked=true, Granted Authorities=[]], org.springframework.security.authentication.UsernamePasswordAuthenticationToken]
2022-08-25 10:37:29.043 DEBUG 1 --- [tp1944815218-18] c.a.d.o.s.p.CloudUserDetailsServiceImpl  : User found [1, admin, admin, null]
2022-08-25 10:37:29.043 DEBUG 1 --- [tp1944815218-18] c.a.d.authentication.AuthenticatedUser   : AuthenticatedUser created [admin, null]
2022-08-25 10:37:29.104 DEBUG 1 --- [tp1944815218-18] c.a.d.o.server.lockout.LockoutService    : Successful Login Event for [admin, org.springframework.security.authentication.UsernamePasswordAuthenticationToken]
2022-08-25 10:37:29.107 DEBUG 1 --- [tp1944815218-18] .a.d.o.s.t.DataCloudAccessTokenConverter : Enhancing access_token for: admin
2022-08-25 10:37:29.118 DEBUG 1 --- [tp1944815218-18] .a.d.o.s.t.DataCloudAccessTokenConverter : refresh_token: TXXIG57Kjmnf3xTOxEMdAiQXYl4
2022-08-25 10:37:29.119 DEBUG 1 --- [tp1944815218-18] .a.d.o.s.t.DataCloudAccessTokenConverter : Enhancing refresh_token for: admin
2022-08-25 10:37:29.120 DEBUG 1 --- [tp1944815218-18] .a.d.o.s.t.DataCloudAccessTokenConverter : refresh_token exp: Thu Aug 25 20:37:29 GMT 2022
2022-08-25 10:37:29.133 DEBUG 1 --- [tp1944815218-19] c.a.d.o.s.controller.LoginController     : Auth response: 200 OK
2022-08-25 10:37:29.137 DEBUG 1 --- [tp1944815218-19] c.a.d.o.s.controller.LoginController     : User: 1
2022-08-25 10:37:29.149 DEBUG 1 --- [tp1944815218-19] c.a.d.api.tools.ServiceCallInterceptor   : POST /api/login 200 OK [132 ms] []

**Comment by auswells** [comment]
This ingress route should also be flipped from base to authorization:
/.well-known/openid-configuration

**Review by auswells — APPROVED** [review]

**Review by manojkumarbardhan — APPROVED** [review]

**Pull request merged**